src/lib.rs

```diff
@@ -87,8 +78,9 @@ pub mod utils;
 
 pub use address::Address;
 pub use error::Error;
-pub use num256::Uint256;
 pub use private_key::PrivateKey;
 pub use signature::Signature;
 pub use transaction::Transaction;
 pub use types::BigEndianInt;
+pub use u64_array_bigints::u256;
```

Comment:
this is a bit confusing u256 vs U256 ?

Reply:
I don't think so. The `u256` macro can't be used without a `!`, and the type system guards against using a macro and a type in the wrong places

Reply:
that does not change the fact that the naming is confusing